src/IssueViz.UnitTests/Editor/QuickActions/IssueLocationActionsSourceTests.cs

```diff
@@ -149,6 +150,33 @@ public void OnTagsChanged_HasSubscribersToSuggestedActionsChanged_RaisesSuggeste
             eventHandler.Verify(x => x(It.IsAny<object>(), It.IsAny<EventArgs>()), Times.Once);
         }
 
+        [TestMethod]
+        public void OnTagsChanged_RunsOnUIThread()
+        {
```

Comment:
Most of this code is copied from other test methods, I've only added the threadhandling bits.